src/components/main/blockchain/blocks/ObjectsSummary.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+import Value from "./Value";
+
+export default function ObjectsSummary({op, listItem}) {
+    return <div>
+        <p>Objects created: <Value listItem={listItem}>{op.new ? op.new.length : 0}</Value></p>
```

Comment:
Should be optional
Here all 3 lines are always visible, shouldn't be visible if 0